src/test_typing.py

```diff
@@ -625,6 +625,19 @@ class MM1(MutableMapping[str, str], collections_abc.MutableMapping):
             class MM2(collections_abc.MutableMapping, MutableMapping[str, str]):
                 pass
 
+    def test_orig_bases(self):
+        T = TypeVar('T')
+        class C(typing.Dict[str, T]): ...
+        self.assertEqual(C.__orig_bases__, (typing.Dict[str, T],))
```

Comment:
I know there are no docs for `__parameters__` etc. either, but I would still like to see some kind of example code that takes a class like this and recovers what it means given the various new dunder methods (`__origin__`, `__orig_bases__`, `__parameters__`). E.g. if I had a dict `{x: y}` how would I check that it's a subclass of `Dict[str, T]`? (The answer should reveal that the type of x must be str and T must be the type of y.)